hl-todo.el

```diff
@@ -420,11 +419,11 @@ current line."
   (interactive
    (list (completing-read
           "Insert keyword: "
-          (cl-mapcan (pcase-lambda (`(,keyword . ,face))
-                       (and (equal (regexp-quote keyword) keyword)
-                            (list (propertize keyword 'face
-                                              (hl-todo--combine-face face)))))
-                     hl-todo-keyword-faces))))
+          (mapcan (pcase-lambda (`(,keyword . ,face))
```

Comment:
`mapcan` was added in 26.1, I believe. We still support 25.

Reply:
mapcan is supported via Compat.

Reply:
doh. Of course.